mocks/handlers.ts

```diff
@@ -0,0 +1,88 @@
+import { rest } from 'msw';
+
+import { EventType, ActivityTypeType } from '../model/Event';
+
+const event: EventType[] = [
+  {
+    id: '60333292-7ca1-4361-bf38-b6b43b90cb16',
+    title: 'LOLO',
```

Comment:
This does not seem like production data. Change it to something less "hovnohovno".